tool/embedded-protocol.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {execSync} from 'child_process';
+import {promises} from 'fs';
```

Comment:
```suggestion
import {promises as fs} from 'fs';
```

This'll get you cleaner-looking invocation sites :).

Reply:
Thanks!